client/faucet/src/mint.rs

```diff
@@ -107,7 +107,18 @@ impl Service {
     }
 
     pub async fn process(&self, mut params: MintParams) -> Result<Response> {
+        println!("process started {}", params);
         let (tc_seq, dd_seq, receiver_seq) = self.sequences(params.receiver()).await?;
+        let mut domain_map = HashMap::new();
+        if let (Some(_diem_id_domain), Some(_is_remove_domain)) =
```

Comment:
only set up when there is "is_remove_domain"? it is possible we get `_diem_id_domain` set, but no `_is_remove_domain`